src/mmw/apps/geoprocessing_api/calcs.py

```diff
@@ -226,8 +231,10 @@ def point_source_pollution(geojson):
             point_source_results = []
 
     return {
-        'displayName': 'Point Source',
-        'name': 'pointsource',
+        'displayName': 'Point Source ({})'.format(datasource.upper())
```

Comment:
f-strings are now available for this project

```suggestion
        'displayName': f'Point Source ({datasource.upper()})'
```